src/main/java/WoopAI/DeadlineTask.java

```diff
@@ -0,0 +1,79 @@
+package WoopAI;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+/**
+ * A class for tasks of deadline type.
+ */
+public class DeadlineTask extends Task{
```

Comment:
```suggestion
public class DeadlineTask extends Task {
```
Coding standard violation